src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -66,11 +66,21 @@ private UserStore(PersistentStorageAgent persistentStorageAgent) {
     users = new ArrayList<>();
 
     // hard-coded initial Admin:
-    // TODO(JW): Merge "the creation process" from RegisterServlet to here.
-    String hashedPassword = BCrypt.hashpw("AdminPass01", BCrypt.gensalt());
-    User initialAdmin = new User(UUID.randomUUID(), "Admin01", hashedPassword, Instant.now());
-    initialAdmin.setAdmin(true);
-    users.add(initialAdmin);
+    this.addUser("Admin01", "AdminPass01", true);
+  }
+
+  /**
+   * Add a new user to the current set of users known to the application. This should only be called
+   * to add a new user, not to update an existing user.
+   */
+  public void addUser(String username, String password, boolean adminStatus) {
```

Comment:
It might be clearer if the parameter were just called `admin` since it would mirror the name of the field on `User`.

Reply:
Agree, fixed!